operations/mimir-mixin/dashboards/queries.libsonnet

```diff
@@ -334,6 +334,17 @@ local filename = 'mimir-queries.json';
         $.panel('Index-header lazy load duration') +
         $.latencyPanel('cortex_bucket_store_indexheader_lazy_load_duration_seconds', '{%s}' % $.jobMatcher($._config.job_names.store_gateway)),
       )
+      .addPanel(
+        $.panel('Index-header lazy load gate latency') +
+        $.queryPanel(
+          |||
+            sum by(stage) (rate(cortex_bucket_stores_gate_duration_seconds_sum{gate="index_header", %s}[$__rate_interval]))
+            /
+            sum by(stage) (rate(cortex_bucket_stores_gate_duration_seconds_count{gate="index_header", %s}[$__rate_interval]))
+          ||| % [$.jobMatcher($._config.job_names.store_gateway), $.jobMatcher($._config.job_names.store_gateway)],
+          'average load latency'
+        )
```

Comment:
i'm not sure why this PR wasn't updated, but nick asked me to change the queryPanel to a latencyPanel since it will include percentiles and an average. i added the descriptions though!